temporalio/worker/workflow_sandbox/in_sandbox.py

```diff
@@ -0,0 +1,62 @@
+"""Code that runs inside the workflow sandbox.
+
+.. warning::
+    This API for this module is considered unstable and may change in future.
+"""
+
+import dataclasses
+import logging
+from typing import Type
+
+import temporalio.bridge.proto.workflow_activation
+import temporalio.bridge.proto.workflow_completion
+import temporalio.worker.workflow_instance
+
+logger = logging.getLogger(__name__)
+
+# Set to true to log lots of sandbox details
+LOG_TRACE = False
```

Comment:
nit: merge this with the other LOG_TRACE var?

Reply:
Which one? I think these should remain file-specific